### ISSUE TITLE
onEnd is not called

### ISSUE THREAD

**Issue by PhilippKrone** [TheWidlarzGroup/react-native-video#61]
Hi,

onEnd seems not to get called when playing from an URL:

``` javascript
                        <Video source={{uri: ITUNES_JSON.previewUrl}} // Can be a URL or a local file.
                           rate={1.0}                   // 0 is paused, 1 is normal.
                           volume={1.0}                 // 0 is muted, 1 is normal.
                           muted={false}                // Mutes the audio entirely.
                           paused={!this.state.itunesPlaying}               // Pauses playback entirely.
                           resizeMode="cover"           // Fill the whole screen at aspect ratio.
                           repeat={false}                // Repeat forever.
                           onEnd={() => {console.log('test');}}/>
```

with the following URL:

``` javascript
  "previewUrl":"http://a1471.phobos.apple.com/us/r2000/004/Music/ec/88/d0/mzm.fqrwrunh.aac.p.m4a",
```

Any idea?

**Comment by isair** [comment]
I did some quick debugging and it seems like `notifyEnd:` is not being called. However, `attachListeners` is.

**Comment by brentvatne** [comment]
Should be fixed by @isair in #62!

**Comment by PhilippKrone** [comment]
@isair @brentvatne big thanks to both of you! Thanks a lot for this fast bugfix :)
